scripts/gen_install.sh

```diff
@@ -0,0 +1,9 @@
+#/bin/sh
```

Comment:
Typo: this a comment instead of a shebang :-)

Reply:
Whoops!!

Reply:
Obviously, I meant it to be a Sixth Edition Unix Shell Script 😉